yb-voyager/src/dbzm/config.go

```diff
@@ -139,3 +142,38 @@ func (c *Config) WriteToFile(filePath string) error {
 	}
 	return nil
 }
+
+// read config file DEBEZIUM_CONF_FILEPATH into a string
+func readConfigFile() (string, error) {
+	configFile, err := os.ReadFile(DEBEZIUM_CONF_FILEPATH)
+	if err != nil {
+		return "", fmt.Errorf("failed to read config file %s: %w", DEBEZIUM_CONF_FILEPATH, err)
+	}
+
+	return string(configFile), nil
+}
+
+// generate/fetch the value for 'debezium.source.database.server.id' property for MySQL
+func getDatabaseServerID() int {
+	databaseServerId := rand.Intn(100000)
+	configFile, err := readConfigFile()
+	if err != nil {
+		log.Fatalf("failed to read config file: %v", err)
+		return databaseServerId
```

Comment:
Add a comment stating it is unreachable stmt. Consider returning `-1`. Same applies another similar error handler.